yt/frontends/enzo_p/data_structures.py

```diff
@@ -295,6 +338,44 @@ def _parse_parameter_file(self):
         self.periodicity = \
           ensure_tuple(np.ones(self.dimensionality, dtype=bool))
 
+        pfn = self.parameter_filename
+        lcfn = pfn[:pfn.rfind(self._suffix)] + ".libconfig"
```

Comment:
Is there a reason to use `rfind` instead of stripping off `len(self._suffix)` ?

Reply:
Only because I assumed `self.parameter_filename` should preserve the original argument given to `yt.load`. If that's not the case, then I can change this.

Reply:
It should. I think I think Matt was saying to do that on `pfn`, not `self.parameter_filename` itself.